src/bans/mod.rs

```diff
@@ -194,6 +194,8 @@ pub fn check(
     output_graph: Option<Box<OutputGraph>>,
     sender: crossbeam::channel::Sender<Pack>,
 ) {
+    let wildcard = VersionReq::parse("*").expect("Parsing wildcard mustnt fail");
```

Comment:
```suggestion
    let wildcard = VersionReq::any();
```

Reply:
hmm, it turns out `VersionReq::any()` and `VersionReq` with `Wildcard(major)` predicate fail equality comparison

Reply:
Hmm ok...nvm then!